src/sentry/static/sentry/app/components/selectInput.jsx

```diff
@@ -32,13 +32,15 @@ const SelectInput = React.createClass({
     /*below is a hack for a bug in edge related to form submitting.
     see: https://github.com/facebook/react/issues/7655
     (@maxbittker)*/
-    let selectedIndex = this.refs.select.selectedIndex;
-    if (this.refs.select && selectedIndex >= 0) {
-     let options = this.refs.select.options;
-     let tempIndex = (selectedIndex + 1) % options.length;
+    if(this.refs.select){
```

Comment:
Hmm, should be two spaces here:

```javascript
if (this.refs.select) {
```

My bad, this should be enforced via eslint ([keyword-spacing](http://eslint.org/docs/rules/keyword-spacing)).